blog/_posts/2018-07-07-brazilian-team.md

```diff
@@ -0,0 +1,78 @@
+I was also a coach on the 2017th edition and it was an amazing experience. The other coach of my group introduced me to the CTO of a Brazilian fintech where later I did a brief software internship before moving to Vancouver. So imagine how happy I was when Ana and Vaishali surprised us announcing that we were selected, Rails Girls has been playing an important role in my tech career!
+
+#### Camila
+
+Hello everybody, my name is Camila! I'm a Brazilian living in Vancouver with my husband. After finishing our master's degree in Computer Science last year we came to Canada with our two cats - the flight was a great adventure!
+
+During graduation I worked with Recommender Systems, characterizing the consumption over time in recommendation domains, and in social network analysis. In my master I was interested in understanding how sociological aspects are reflected in current technologies, such as the existence of racial bias/stereotypes in search engines, and what is the role of the algorithms on the propagation of these bias/stereotypes. I've been working with academic research since my first year as a undergrad student. And my plan was to finish my master's degree and then apply for a PhD. But for now I feel the need to apply my knowledge to something that is really going to be used by several people and not only in the academic environment. 
+
+I learned to code in my early years of graduation and I have a good experience with algorithms. I have good knowledge in Python, C, R... But I have almost no experience in software development, software engineering, testing, etc. So I think the Rail Girls program will be a great opportunity to be part of the Open Source community and learn a lot!
+
+### How did we meet?
+
+Because we are both relatively new to Vancouver, it was difficult to find a teammate here. So we used the RGSoC google groups to search for teammates and there was where we met each other. We went to a coffee shop and discussed about our application and we soon got along, especially because we both have cats =).
+
+Thanks, RGSoc organizers for providing these mediums for us to find a teammate :)
+
+### How is work going?
+
+This week has been great! We are reading and collecting information from the Public Lab codebase and the API docs/tests. Also we focused on learning more about Git, RESTful API’s, and how to document code changes.
+
+We also want to thank the [Predictable Revenue](https://predictablerevenue.com/) for letting us use their space to work. Thiago, one of our coaches, works there and we asked if we could work there. The office is great and it’s located at the historical site of Vancouver, a central location in the city.
+
+![Brazilian Housewives Coding Team](/img/blog/2018/2018-07-09-brazilian-housewives-coding-team.jpg)
+<div class="image-credits"><b>Camila, Julio, Stefanni and Thiago on our first day! picture taken by Stefanni Brasil</b></div>
+
+### Meet our Amazing Team
+
+** Coaches: ** 
```

Comment:
Did you intend to put these titles [coaches, mentor etc] in bold? Then you'll need to remove the space between asterisks and text. If not, then it's fine.

Also, for the consistency purpose either use all titles with colon or without colon.